uipa_org/settings/base.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+from configurations import values
+from froide.settings import Base
+import os
+import re
+from pathlib import Path
+
+rec = lambda x: re.compile(x, re.I | re.U)
+
+def env(key, default=None):
+    return os.environ.get(key, default)
+
+THEME_ROOT = Path(__file__).resolve().parent.parent
+
+# Note: The settings in this file are either Django settings or Froide settings.
+# Django settings are documented at https://docs.djangoproject.com/en/4.2/ref/settings/ (or the version of Django you are using).
+# Froide settings are documented at https://froide.readthedocs.io/en/latest/configuration/
+
+class UipaOrgThemeBase(Base):
+    FROIDE_THEME = 'uipa_org.theme'
+
+    SITE_NAME = "UIPA.org"
+    SITE_EMAIL = "info@uipa.org"
+    SITE_URL = 'http://localhost:8000'
+    
+    FRONTEND_BUILD_DIR = THEME_ROOT.parent / "build"
+    STATIC_ROOT = values.Value(THEME_ROOT.parent / "public")
+
+    @property
+    def STATICFILES_DIRS(self):
+        return [THEME_ROOT / "uipa_org/theme/static"] + super().STATICFILES_DIRS
```

Comment:
Should be "theme/static" to eliminate the warning about "STATICFILES_DIRS setting does not exist". Then can drop "--skip-checks" from the "python manage.py migrate" command.